src/main/java/Bob/Bob.java

```diff
@@ -0,0 +1,18 @@
+package bob;
+
+import bob.task.Task;
+
+import java.util.ArrayList; 
+import java.util.List;
+import java.util.Scanner;
```

Comment:
```suggestion
import java.util.ArrayList; 
import java.util.List;
import java.util.Scanner;

import bob.task.Task;
```

The order of imports in this file seem to differ from the rest. Typicall you did
```java
import java.____

import bob.____
```
Perhaps you can change the import order to be consistent with the rest of the files?